spyder/plugins/editor/plugin.py

```diff
@@ -52,7 +52,7 @@
     SelectionContextModificator, ExtraAction)
 from spyder.plugins.editor.confpage import EditorConfigPage
 from spyder.plugins.editor.utils.autosave import AutosaveForPlugin
-from spyder.plugins.editor.utils.switcher import EditorSwitcherManager
+# from spyder.plugins.switcher.manager import EditorSwitcherManager
```

Comment:
We should uncomment this for the suggestion bellow to work:
```suggestion
from spyder.plugins.switcher.manager import EditorSwitcherManager
```